### PR TITLE
Added PyPI rss update stream ingestor

### PR DESCRIPTION
**Pull request by jsonperl**
This is the updates bit of the ingestion. I'm going to add the new packages bit next, but since it didn't include a version, I'm going to get a little more complex and call out to each of the items to get the versions available before sending them off to the publisher.

I'm sneaking in the removal of the npm migration thing from the last PR, it's pretty innocuous.

### CODE REVIEW

File: ingestors/npm.go [hunk 1]
```diff
@@ -39,16 +38,11 @@ func (ingestor *NPM) Name() string {
 }
 
 func (ingestor *NPM) Ingest(results chan data.PackageVersion) {
-	since, err := getBookmark(ingestor, nil)
```
Comment (katzj) [inline ingestors/npm.go:42]: Can you at least split into a separate commit? :)

**Review by katzj — COMMENTED** [review]
Do new packages not show up in updates?

**Comment by jsonperl** [comment]
> Do new packages not show up in updates?

That would have been nice huh!

File: ingestors/npm.go (hunk 1 above)
Comment (jsonperl) [reply ingestors/npm.go:42]: Yes, haha. I'll break out another PR.

File: ingestors/pypi_rss.go [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	"time"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/librariesio/depper/data"
+	"github.com/mmcdole/gofeed"
+)
+
+//const pyPiPackagesFeedUrl = "https://pypi.org/rss/packages.xml"
+const pyPiUpdatesFeedUrl = "https://pypi.org/rss/updates.xml"
```
Comment (tiegz) [inline ingestors/pypi_rss.go:14]: Packagist had both feeds as well, but I ditched the packages feed because new packages would show up in the new releases feed anyway. For instance, this Pypi package with 1 version just showed up in the above feed: https://pypi.org/project/nevopy/0.0.1/#history 

Should we just skip the new packages feed?

File: ingestors/pypi_rss.go [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+type PyPiRss struct {
```
Comment (tiegz) [inline ingestors/pypi_rss.go:16]: Any reason not to call it `PyPi`?

**Review by katzj — APPROVED** [review]

File: ingestors/pypi_rss.go (hunk 3 above)
Comment (jsonperl) [reply ingestors/pypi_rss.go:16]: Because we're going to have two

Comment (jsonperl) [reply ingestors/pypi_rss.go:16]: https://app.clubhouse.io/tidelift/story/15803/build-a-xml-rpc-pypi-ingestor-for-depper

File: ingestors/pypi_rss.go (hunk 2 above)
Comment (jsonperl) [reply ingestors/pypi_rss.go:14]: Hmm I've been checking pretty regularly and haven't seen them in the updates feed, will poke some more.

Comment (jsonperl) [reply ingestors/pypi_rss.go:14]: <item>
      <title>sshcmd added to PyPI</title>
      <link>https://pypi.org/project/sshcmd/</link>
      <guid>https://pypi.org/project/sshcmd/</guid>
      <description>ssh cmd</description>
      <author>xliang.lee@qq.com</author>
      <pubDate>Tue, 02 Feb 2021 10:27:12 GMT</pubDate>
    </item>
  </channel>
</rss>

JP:~/Code/tidelift/depper$ curl https://pypi.org/rss/updates.xml | grep sshcmd
=> nothin

Comment (jsonperl) [reply ingestors/pypi_rss.go:14]: Are you positive packagist did a better job than pypi is doing here? Have you run through a couple examples and seen them in both feeds?

Comment (tiegz) [reply ingestors/pypi_rss.go:14]: > Are you positive packagist did a better job than pypi is doing here? Have you run through a couple examples and seen them in both feeds?

Packagist could possibly return new packages that don't have versions yet ([here's the query](https://github.com/composer/packagist/blob/dc36ab17a66f2af14f77dd1273b88c12d82f0f3b/src/Entity/PackageRepository.php#L504)), but I figured we don't really care about a package until it has releases. Wondering what you and @katzj think about that tho.

Comment (tiegz) [reply ingestors/pypi_rss.go:14]: But here's an example I just saw in pypi feeds:

* from `updates.xml`: `multiplespawner 0.0.0a0` (Tue, 02 Feb 2021 15:31:25 GMT)
* from `packages.xml`: `multiplespawner added to PyPI` (Tue, 02 Feb 2021 15:31:25 GMT)

Comment (tiegz) [reply ingestors/pypi_rss.go:14]: I also don't think there's a way to "Create a new project" on Pypi, so the only way to create one is to release it via twine, which would require you to have a version set anyway. 

All that said, probably can't hurt if you still wanna do the packages feed still 🤷

Comment (jsonperl) [reply ingestors/pypi_rss.go:14]: I guess the question is can we _count on_ the updates feed having the versions from the new packages. I haven't been able to prove that to myself as yet.

Comment (tiegz) [reply ingestors/pypi_rss.go:14]: I think they're just pulling releases from all projects, ordered by release created_at: https://github.com/pypa/warehouse/blob/main/warehouse/rss/views.py#L66-L72
